scripts/globals/npc_util.lua

```diff
@@ -435,15 +435,20 @@ function npcUtil.completeRecord(player, record, params)
     end
 
     if params["xp"] ~= nil and type(params["xp"]) == "number" then
-        player:addExp(params["xp"] * EXP_RATE)
+        player:addExp(params["xp"] * ROE_EXP_RATE)
     end
 
     if params["keyItem"] ~= nil then
         npcUtil.giveKeyItem(player, params["keyItem"])
     end
 
     -- successfully complete the record
-    player:setEminenceCompleted(record)
+    if params["repeatable"] == true then
```

Comment:
this one of the special cases where we need the explicit comparison?

Reply:
I don't think so. Just a style choice maybe? @Apocarypse

Reply:
Good improvement either way @TeoTwawki . Put it in the recent commit.